python/python/ert/enkf/node_id.py

```diff
@@ -17,4 +18,4 @@ def __init__(self, report_step, realization_number):
 
 
 CWrapper.registerType("node_id", NodeId)
-
+registerType( "node_id" , NodeId ) 
```

Comment:
I let both old and new be together - for now. Since it does not inherit from BaseCclass I guess new style should be with direct call to `registerType()`.